ui/src/seminars/components/SeminarItem.jsx

```diff
@@ -32,27 +32,29 @@ function SeminarItem({ metadata, selectedTimezone }) {
   return (
     <ResultItem
       leftActions={
-        <>
-          {urls && <UrlsAction urls={urls} />}
-          {joinUrls && (
-            <UrlsAction
-              urls={joinUrls}
-              icon={<VideoCameraAddOutlined />}
-              text="join"
-            />
-          )}
-          {materialUrls && (
-            <UrlsAction
-              urls={materialUrls}
-              icon={<FileOutlined />}
-              text="material"
-            />
-          )}
-          <ExportToCalendarAction seminar={metadata} />
-          {canEdit && (
-            <EditRecordAction pidType="seminars" pidValue={recordId} />
-          )}
-        </>
+        enableActions && (
```

Comment:
problem with this approach (just hiding with flag), we deliver more data, and do more serializing than we need.. but I guess this is not significant at this point.